spec/models/importers/all_taxons_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.describe Importers::AllTaxons do
+  describe '#run' do
+    it 'creates a new Taxonomy if not present' do
+      attrs1 = { content_id: 'a-content-item_-1', title: 'a-title-1' }
+      attrs2 = { content_id: 'a-content-item_-2', title: 'a-title-2' }
+      subject.taxonomies_service = double
+
+      allow(subject.taxonomies_service).to receive(:find_each).and_yield(attrs1).and_yield(attrs2)
+
+      expect { subject.run }.to change { Taxonomy.count }.by(2)
+    end
+
+    context 'when the taxonomy exists' do
+      before { create :taxonomy, title: 'old-title' }
+      it 'updates the attributes' do
+        attrs1 = { content_id: 'a-content-id-1', title: 'a-title-1' }
+        subject.taxonomies_service = double
+
+        allow(subject.taxonomies_service).to receive(:find_each).and_yield(attrs1)
+
+        subject.run
+
+        attributes = Taxonomy.find_by(content_id: 'a-content-id-1').attributes.symbolize_keys
+        expect(attributes).to include(content_id: 'a-content-id-1', title: 'a-title-1')
```

Comment:
1. Not sure if you are updating the taxonomy as the content_id could not be the same
2. Why not asserting only the attribute that is being updated?

```ruby
Taxonomy.find_by(content_id: 'a-content-id-1')
expect(taxonomy.title).to eq('a-title-1')
```